main.py

```
@@ -38,14 +39,28 @@
 from .core.angel_heart_context import AngelHeartContext
+from .core.chat_profile import ChatProfileStore
 from .core.runtime_task_tracker import RuntimeTaskTracker, track_runtime_handler
 from .tools.image_understanding import AngelDescribeImageTool
 
 # 在框架加载 schema 之前执行配置迁移
 run_migration()
 
 
-@register("astrbot_plugin_angel_heart", "kawayiYokami", "天使心秘书，让astrbot拥有极其聪明，有分寸的群聊介入，和极其完备的群聊上下文管理", "0.8.11", "https://github.com/kawayiYokami/astrbot_plugin_angel_heart")
+def _plugin_version() -> str:
+    """从 metadata.yaml 读取版本号，避免与 @register 双处维护。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)